indico/modules/events/registration/util.py

```diff
@@ -137,6 +138,9 @@ def get_flat_section_submission_data(regform, *, management=False, registration=
             continue
         if registration and isinstance(item.field_impl, (ChoiceBaseField, AccommodationField)):
             field_data = get_field_merged_options(item, registration_data)
+        elif registration and isinstance(item.field_impl, AccompanyingPersonsField):
+            field_data = item.view_data
+            field_data['availablePlaces'] = item.field_impl.get_available_places(False)
```

Comment:
I *think* we camelize the dict later, so if that's indeed the case you should be able to use the `snake_cased` version here.